front/src/common/UserSettings.tsx

```diff
@@ -55,6 +58,28 @@ const UserSettings = () => {
         <small id="safeWordHelpBlock" className="form-text text-muted">
           {t('safeWordHelp')}
         </small>
+        {
+          // TODO PACEDTRAIN: Remove switch after development pacedTrain feature
+          isSuperUser && (
+            <div className="mt-3 d-flex align-items-center mt-2">
```

Comment:
You have mt-3 and mt-2 here, I think 1 of them is enough

```suggestion
            <div className="mt-3 d-flex align-items-center mt-2">
```

Reply:
Nice, catch thank you

Reply:
Done